packages/devtools/lib/src/timeline/timeline_protocol.dart

```diff
@@ -322,7 +322,12 @@ class TimelineFrame {
   final String id;
 
   /// Marks whether this frame has been added to the timeline.
```

Comment:
document that this can only be set once.

Reply:
done.